package.json

```diff
@@ -37,46 +37,48 @@
 		"@wordpress/browserslist-config": "^2.1.2",
 		"@wordpress/scripts": "^1.1.3",
 		"assets-webpack-plugin": "^3.5.1",
-		"autoprefixer": "^8.2.0",
+		"autoprefixer": "^8.4.1",
 		"babel-core": "^6.26.3",
 		"babel-eslint": "^8.2.2",
 		"babel-loader": "^7.1.2",
 		"babel-plugin-transform-async-generator-functions": "^6.24.1",
 		"babel-plugin-transform-react-jsx": "^6.24.1",
 		"babel-polyfill": "^6.26.0",
 		"clean-webpack-plugin": "^0.1.18",
-		"codecov": "^3.0.1",
+		"codecov": "^3.0.2",
 		"concurrently": "^3.5.1",
-		"core-js": "^2.5.5",
-		"cross-env": "^5.1.4",
+		"core-js": "^2.5.6",
+		"cross-env": "^5.1.5",
 		"css-loader": "^0.28.11",
 		"deep-freeze": "0.0.1",
 		"eslint": "^4.19.1",
 		"eslint-config-wordpress": "^2.0.0",
 		"eslint-plugin-jest": "^21.15.1",
 		"eslint-plugin-jsx-a11y": "^6.0.3",
-		"eslint-plugin-react": "^7.7.0",
+		"eslint-plugin-react": "^7.8.1",
 		"eslint-plugin-wordpress": "git://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress.git#1774343f6226052a46b081e01db3fca8793cc9f1",
 		"extract-text-webpack-plugin": "^3.0.2",
-		"gutenberg": "Wordpress/gutenberg#master",
+		"gutenberg": "Wordpress/gutenberg#v2.8.0",
```

Comment:
This is a workaround for the [issue I posted here](https://github.com/WordPress/gutenberg/issues/6718).  By pinning gutenberg to a specific tag, this will prevent the errors from happening that killed our js test runs.  This will fix what's happening in the `Gutenberg/master` branch once this branch is merged back into it.